OpenOversight/migrations/versions/cd39b33b5360_constrain_officer_gender_options.py

```diff
@@ -0,0 +1,73 @@
+"""constrain officer gender options
+
+Revision ID: cd39b33b5360
+Revises: 562bd5f1bc1f
+Create Date: 2020-07-13 02:45:07.533549
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'cd39b33b5360'
+down_revision = '562bd5f1bc1f'
+branch_labels = None
+depends_on = None
+
+def get_update_statement(normalized, options):
+    return f"""
+    UPDATE officers 
+    SET gender = '{normalized}'
+    WHERE gender::citext in ({', '.join(["'" + o + "'" for o in options])});
```

Comment:
`gender::citext` just means the match is case-insensitive

Reply:
so this generates statements that say, basically, "set gender to M if the existing gender is 'male', 'm', or 'man' and ignore case" (but for each gender option)